### PR TITLE
feat(react): Adds FieldWrap component

### PR DESCRIPTION
**Pull request by schne324**

### CODE REVIEW

File: docs/patterns/components/TextField/index.js [hunk 1]
```diff
@@ -1,97 +1,109 @@
...
-            description: 'Custom "required" text. Useful for localization.'
-          },
-          multiline: {
-            type: 'boolean',
-            description:
-              'If true, a textarea element will be rendered. Otherwise, an input[type="text"] will be rendered.'
-          }
-        }}
-      />
+      <FieldWrap>
```
Comment (schne324) [inline docs/patterns/components/TextField/index.js:11]: This should resolve a few of the a11y issues in the e2e tests #386

File: packages/styles/panel.css [hunk 2]
```diff
@@ -12,14 +12,26 @@
...
   font-weight: var(--font-weight-bold);
 }
 
+fieldset.Panel {
+  position: relative;
+  padding-top: 76px;
+}
+
+/* reset absurd browser-default legend styles */
+.Panel legend {
```
Comment (schne324) [inline packages/styles/panel.css:30]: I've made it so the legend matches the exact styling of the Panel__Heading elements

File: docs/patterns/components/Checkbox/index.js [hunk 3]
```diff
@@ -21,80 +22,84 @@ export default class CheckboxDemo extends Component {
...
-            description:
-              'If the checkbox should be checked, which allows it to be "controlled"'
-          },
-          disabled: {
-            type: 'boolean',
-            description: 'If the checkbox should be disabled'
-          }
-        }}
-      />
+      <FieldWrap>
```
Comment (schne324) [inline docs/patterns/components/Checkbox/index.js:25]: resolves a11y issues in #386

File: docs/patterns/components/RadioGroup/index.js [hunk 4]
```diff
@@ -15,7 +16,8 @@ export default class RadioGroupDemo extends Component {
   render() {
     const { value } = this.state;
     return (
-      <div>
+      <FieldWrap>
```
Comment (schne324) [inline docs/patterns/components/RadioGroup/index.js:19]: resolves a11y issues in #386

File: docs/patterns/components/Select/index.js [hunk 5]
```diff
@@ -21,7 +22,8 @@ const SelectDemo = () => {
   const [currentValue, setCurrentValue] = useState('Maybe');
 
   return (
-    <div className="SelectDemo">
+    <FieldWrap className="SelectDemo">
```
Comment (schne324) [inline docs/patterns/components/Select/index.js:25]: resolves a11y issues in #386

**Review by thuey — APPROVED** [review]
lgtm!

**Pull request merged**